pkg/pmem-csi-operator/controller/deployment/controller_driver.go

```diff
@@ -25,18 +26,52 @@ import (
 	rbacv1 "k8s.io/api/rbac/v1"
 	storagev1beta1 "k8s.io/api/storage/v1beta1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	apiruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/klog"
+	"k8s.io/kubectl/pkg/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	controllerServicePort = 10000
 	controllerMetricsPort = 10010
 	nodeControllerPort    = 10001
+
+	// Annotation used to tag/track the version of the operator that
+	// successfully reconciled a deployment.
+	VersionAnnotation = "pmem-csi.intel.com/operatorVersion"
```

Comment:
This new concept is not clear to me. Why do we need this annotation?

Reply:
It only seems to be used for testing. We shouldn't introduce extra, otherwise unused features into the operator API just for testing purposes.

Reply:
Removed the version annotation.